docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -229,46 +239,3 @@ A playbook would looks like this:
 
 Note that use of the "add_host" module above creates a temporary, in-memory group.  This means that a play in the same playbook can then manage machines
 in the 'new_instances' group, if so desired.  Any sort of arbitrary configuration is possible at this point.
```

Comment:
While you're overhauling the guide, would you like to add any links to GCP documentation?

Reply:
I added the link to the Google Cloud homepage